lib/plaid/identity_verification.ex

```diff
@@ -0,0 +1,589 @@
+      end
+    end
+  end
+
+  defmodule SelfieCheck do
+    @moduledoc """
+    Plaid Identity Verification Selfie Check data structure.
+    """
+
+    @derive Jason.Encoder
+    defstruct status: nil,
+              attempt: nil,
+              capture: [],
+              analysis: nil
+
+    @type t :: %__MODULE__{
+            status: String.t(),
+            attempt: integer(),
+            capture: [Plaid.IdentityVerification.SelfieCheck.Capture.t()],
+            analysis: Plaid.IdentityVerification.SelfieCheck.Analysis.t() | nil
+          }
+
+    defmodule Capture do
+      @moduledoc """
+      Plaid Identity Verification Selfie Capture data structure.
+      """
+
+      @derive Jason.Encoder
+      defstruct url: nil, video_url: nil, capture_method: nil
+
+      @type t :: %__MODULE__{
+              url: String.t() | nil,
+              video_url: String.t() | nil,
+              capture_method: String.t()
+            }
+    end
+
+    defmodule Analysis do
+      @moduledoc """
+      Plaid Identity Verification Selfie Analysis data structure.
+      """
+
+      @derive Jason.Encoder
+      defstruct face_match: nil, image_quality: nil
+
+      @type t :: %__MODULE__{
+              face_match: String.t() | nil,
+              image_quality: String.t() | nil
+            }
+    end
+  end
+
+  defmodule KYCCheck do
+    @moduledoc """
+    Plaid Identity Verification KYC Check data structure.
+    """
+
+    @derive Jason.Encoder
+    defstruct status: nil,
+              address: nil,
+              name: nil,
+              date_of_birth: nil,
+              id_number: nil,
+              phone_number: nil
+
+    @type t :: %__MODULE__{
+            status: String.t(),
+            address: String.t() | nil,
+            name: String.t() | nil,
+            date_of_birth: String.t() | nil,
+            id_number: String.t() | nil,
+            phone_number: String.t() | nil
+          }
+  end
+
+  defmodule RiskCheck do
+    @moduledoc """
+    Plaid Identity Verification Risk Check data structure.
+    """
+
+    @derive Jason.Encoder
+    defstruct status: nil,
+              risk_score: nil,
+              linked_services: [],
+              behavior: nil,
+              email: nil,
+              phone: nil
+
+    @type t :: %__MODULE__{
+            status: String.t(),
+            risk_score: integer() | nil,
+            linked_services: [String.t()],
+            behavior: map() | nil,
+            email: map() | nil,
+            phone: map() | nil
+          }
+  end
+
+  defmodule WatchlistScreening do
+    @moduledoc """
+    Plaid Identity Verification Watchlist Screening data structure.
+    """
+
+    @derive Jason.Encoder
+    defstruct status: nil,
+              audit_trail: nil,
+              search_terms: nil
+
+    @type t :: %__MODULE__{
+            status: String.t(),
+            audit_trail: String.t() | nil,
+            search_terms: map() | nil
+          }
+  end
+
+  defmodule ListResponse do
```

Comment:
Can you please rename this to `Verifications` or similar? The library has a precedent for having domain-named data structures instead of generic ones, e.g. [Payments](https://github.com/wfgilman/plaid-elixir/blob/47df2f4a44c7a248add368828f9cff44a4ddeb1d/lib/plaid/payment_initiation/payments.ex#L1-L21)